pkg/scheduler/plugins/elasticquota/plugin_helper.go

```diff
@@ -0,0 +1,198 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elasticquota
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	apiresource "k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/api/v1/resource"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
+	schedulerv1alpha1 "sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+	"sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+)
+
+// getPodAssociateQuotaName If pod's don't have the "quota-name" label, we will use the namespace to associate pod with quota
+// group. If the plugin can't find the matched quota group, it will force the pod to associate with the "default-group".
+func (g *Plugin) getPodAssociateQuotaName(pod *v1.Pod) string {
+	quotaName := pod.Labels[extension.LabelQuotaName]
+	if quotaName == "" {
+		list, err := g.elasticQuotaController.eqLister.ElasticQuotas(pod.Namespace).List(labels.Everything())
+		if err != nil {
+			runtime.HandleError(err)
+			return extension.DefaultQuotaName
+		}
+		if len(list) == 0 {
+			return extension.DefaultQuotaName
+		}
+		// todo when elastic quota supports multiple instances in a namespace, modify this
+		quotaName = list[0].Name
+	}
+	// can't get the quotaInfo by quotaName, let the pod belongs to DefaultQuotaGroup
+	if g.groupQuotaManager.GetQuotaInfoByName(quotaName) == nil {
+		quotaName = extension.DefaultQuotaName
+	}
+
+	return quotaName
+}
+
+// createDefaultQuotaIfNotPresent create DefaultQuotaGroup's CRD
+func (g *Plugin) createDefaultQuotaIfNotPresent() {
```

Comment:
I also think you need createSystemQuotaIfNotPresent and can't modify anything like max\min in webhook

Reply:
fixed